tomls/omnibus-mainnet/collaterals/wsteth.toml

```diff
@@ -0,0 +1,24 @@
+[invoke.CoreProxy_configureCollateral_wsteth]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"  # "args" see below in the multiline block
+
+[[invoke.CoreProxy_configureCollateral_wsteth.args]]
+tokenAddress = "<%= settings.wsteth_address %>"
+oracleNodeId = "<%= extras.wsteth_oracle_id %>"
+issuanceRatioD18 = "<%= parseEther('2') %>"
+liquidationRatioD18 = "<%= parseEther('1.10') %>"
+liquidationRewardD18 = "<%= parseEther('0.005') %>"
+minDelegationD18 = "<%= parseEther('0.01') %>"
```

Comment:
can we normalise all configs and have settings for `susde`? I can see 3 new collaterals and each has somewhat different implementation. Would be easier if they all follow exact structure